server/src/services/person.service.ts

```diff
@@ -369,6 +366,7 @@ export class PersonService {
           boundingBoxY1: face.boundingBox.y1,
           boundingBoxX2: face.boundingBox.x2,
           boundingBoxY2: face.boundingBox.y2,
+          sourceType: null,
```

Comment:
IMO this doesn't ever need to get set if we are using a default value.

Reply:
Fixed by e9d71c1bcbff3925a5ed2d130d3d3b3011d7a320